File: pset1/mario/more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    
    do
    {
        printf("Height: ");
        height = get_int();
    }while(height < 0 || height >23);
    
    for(int i = 0; i < height; i++)
    {
        for(int j = 0; j < height; j ++)
        {
            if(i + j + 1 < height)
            {
                printf(" ");
            }
            else
            {
                printf("#");
            }
        }
        
        printf("  ");
        
        for(int k = 0; k < height; k++)
        {
            if(k <= i)
            {
                printf("#");
            }
        }
        printf("\n");
    }
}

File: pset6/mario.py
import cs50

def main():
    while True:
        print("Height: ", end="")
        height = cs50.get_int()
        if height>0 and height<23:
            break
        
    for i in range(height):
        for j in range(height):
            if i+j+1 < height:
                print(" ", end="")
            else:
                print("#", end="")
        
        print(" ", end="")
        
        for k in range(height):
            if k <= i:
                print("#", end="")
        
        print()

if __name__ == "__main__":
    main()